pkgs/test_core/lib/src/executable.dart

```diff
@@ -156,6 +156,9 @@ Future<void> _execute(List<String> args) async {
   } on FormatException catch (error) {
     stderr.writeln(error.message);
     exitCode = exit_codes.data;
+  } on FileSystemException {
```

Comment:
This will log more nicely (gives you the path and underlying OS error information)

```suggestion
  } on FileSystemException catch (error) {
    var loadException = LoadException(error.path, error.osError);
```